### PR TITLE
Posts can be liked

### PR DESCRIPTION
**Pull request by BDCraven**

### CODE REVIEW

**Comment by BDCraven** [comment]
Polymorphism not yet implemented. This is the first iteration so a post can have many likes and a like belongs to a post.

**Review by dbugsy — APPROVED** [review]
I really like that you have gone minimal - it works, it is very basic, but it is a proof of concept. We should add some tickets to improve this functionality with javascript - refreshing the page for a like is pretty bad ui.

:shipit: :shipit:

File: spec/helpers/likes_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# describe LikesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe LikesHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (i-hardy) [inline spec/helpers/likes_helper_spec.rb]: This test isn't doing anything- could be deleted!

File: spec/models/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (i-hardy) [inline spec/models/like_spec.rb]: Same for this one!

**Review by i-hardy — APPROVED** [review]
:shipit:

**Review by petrakh — APPROVED** [review]
:shipit:

**Pull request merged**